### PR TITLE
Planet controls

### PR DESCRIPTION
**Pull request by clement-igonet**
Adapt OrbitControls example for planet

### CODE REVIEW

File: examples/js/controls/OrbitControls.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ THREE.OrbitControls = function ( object, domElement ) {
 	// How far you can orbit vertically, upper and lower limits.
 	// Range is 0 to Math.PI radians.
 	this.minPolarAngle = 0; // radians
-	this.maxPolarAngle = Math.PI; // radians
+	this.maxPolarAngle = 11 * Math.PI / 24; // radians
```
Comment (WestLangley) [inline examples/js/controls/OrbitControls.js]: Sorry, this is not acceptable. We can't modify `OrbitControls` to accommodate your use case.

Comment (clement-igonet) [reply examples/js/controls/OrbitControls.js]: OK, I cancel my change in existing OrbitControls.js file in my last commit, and I applied the change in my specific example file misc_controls_earth.js.

**Comment by WestLangley** [comment]
I don't understand the benefit of this example.

Sorry, your monkey-patch of `OrbitControls` does not make sense to me. Perhaps it does to someone else...

**Comment by clement-igonet** [comment]
"monkey-patch" ? I'm not sure to understand what you mean...
Well, I try to do my best to provide an example that shows how to navigate around a planet in a convenient way. Some years ago, I looked at this kind of solution and I did not find any one. So, I thought it was up to me to help the community on that topic.
I suggested a PlanetControls very close to OrbitControls and it was suggested to rather rely on existing OrbitControls and to adapt it (if mandatory) to accept my needs. That's what I've done as I could.
I hope it makes sense to any one.

**Comment by WestLangley** [comment]
`OrbitControls.panLeft()` is supposed to pan a camera. You have modified the method at runtime so the left mouse rotates the earth mesh,  while right mouse rotates the camera. 

Sorry, I don't think this is something that we can accept. If you'd like to share it, why not post it on GitHub?

/ping @Mugen87 for a second opinion.

**Comment by clement-igonet** [comment]
"post it on GitHub"?
I thougth I could contribute to threejs rather than working alone in my unknown project... 😀

What code improvement would you suggest to accept my use case implementation?

**Comment by clement-igonet** [comment]
And also, if we consider "panning" is a move parallel to the ground, can't it be acceptable to keep this name in the use case I suggest?
Not my fault if planets look like spheres! 😀

**Comment by Mugen87** [comment]
As @WestLangley said, the way you are using `OrbitControls` is not coherent. We just try to avoid use-case specific modifications on library entities which should as generic and consistent as possible. So i also suggest to put your code into a gist or own repo. You can use the [forum](https://discourse.threejs.org/) to draw some attention to your work.

**Comment by clement-igonet** [comment]
Ok. So maybe I could at least suggest to replace the current random scene with a plan representation of the earth.
And then I would work again to suggest a solution closer to the real spherical planet use case.

**Pull request closed without merging**